src/Figure.ts

```diff
@@ -21,16 +21,16 @@ export default class Figure extends Builder {
 
     this.isInformative = node.hasAttribute('informative');
     this.captionElem = node.querySelector('emu-caption');
-    this.caption = this.type.charAt(0).toUpperCase() + this.type.slice(1) + ' ' + this.number;
+    this.caption = `${this.type.charAt(0).toUpperCase()}${this.type.slice(1)} ${spec._figurePrefix}${this.number}`;
 
     if (this.isInformative) {
       this.caption += ' (Informative)';
     }
 
     if (this.captionElem) {
-      this.caption += ': ' + this.captionElem.innerHTML;
+      this.caption += ' — ' + this.captionElem.innerHTML;
```

Comment:
```suggestion
      this.caption += ' \u2014 ' + this.captionElem.innerHTML;
```

To make it a little more clear that this isn't an ordinary hyphen. Same below.